addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2695,6 +2697,13 @@ var SnippetsMenu = Widget.extend({
                 $snippet = $globalSnippet;
             }
         }
+        if (this.options.enableTranslation && $snippet && !this._allowInTranslationMode($snippet)) {
+            // In translate mode, only activate allowed snippets (e.g., even if
+            // we create editors for invisible elements when translating them,
+            // we only want to toggle their visibility when the related sidebar
+            // buttons are clicked).
+            return;
+        }
```

Comment:
I'll merge it as it now as the fix is urgent but why is this not needed in older versions @xO-Tx ?

Reply:
@qsm-odoo actually, [it was already there before allowing text options translation](https://github.com/odoo/odoo/pull/158352/commits/242ea80d31e948612ad0328a24063c4336031790#diff-52a4f9d2c217548e69e6b7fd097f286f1754a6389734eea254b87255e501cbefL2658-L2663), but since we exceptionally allow the creation of editors for invisible elements, it needs to be restored again :+1: